### PR TITLE
Transfer of Presences from Bryan to Myself 

### LINKED ISSUE #2628: Amazon Music presence no longer working
**Describe the bug**
Presence in question: https://premid.app/store/presences/Amazon%20Music

When streaming music on Amazon Music, the presence no longer picks up what is being played. Amazon updated their web interface/look which seems to have broken it a few weeks ago.

**To Reproduce**
1. Go to https://music.amazon.com/
2. Play any song

**Expected behavior**
Previously the presence would display in real time whatever song was playing. Now it shows nothing.

**Screenshots**
https://i.imgur.com/W53fZpQ.png

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 87

### LINKED ISSUE #2802: Amazon Music Presence Does Not Work
**Describe the Bug**
When using the Amazon Music presence it does not show anything as your Discord status when listening to Amazon Music.

**Steps to Reproduce**
1. Enable the Amazon Music presence
2. Listen to a song on Amazon Music
3. Look at your Discord status

**Expected Behavior**
The song you are listening to should show as your Discord status along with the time left. It should also show that you are listening on Amazon Music.

**Environment**

Windows 10
Microsoft Edge
Version 87.0.664.75 (Official build) (64-bit)

### LINKED ISSUE #2914: Amazon Extension Doesn't work
When I added the extension it simply didn't show anything in discord but when using the others it did show me their respective states.

_1.  I headed to the preMID store.
2.  I looked for Amazon Music Extension.
3.  I started adding the extension. 
4.  I opened both the application and the web page but nothing happened on Discord._

I expected it to show the status as it was doing in the others.

**Screenshots**
----------------------------------
https://imgur.com/a/98mgPdG
----------------------------------

- OS: Windows 10 Pro.
- Browser: Microsoft Edge.
- Version: 88.0.705.56 (64 bits).

### PR DESCRIPTION
**Pull request by StrikerFRFX**
This PR also includes another change to a different Presence owned by me, Metastream (just fixing Codacy issues) which I forgot in my previous PR lel (#2955)

The list of Presences transferred to myself are as follows:

- Skribbl.io
- Gmail
- Gartic.io & Gartic.com.br
- MangaDex
- Wattpad
- Amazon Music
- Anime Digital Network
- Pokémon Showdown
- Vimeo
- iHeartRadio

The rest of the Presences which I have not transferred will be taken by @snoweyfoxx (and a few by @Bas950 as well)

All Logos and Thumbnails have been updated since the older Gyazo links do not resolve, and I have bumped versions as well. I'll add broken Presences needed bugfixes (referencing Issues) to my TODO List and they will be gradually fixed over the next few weeks. New applications have also been created and assets have been uploaded too (and IDs changed).

Any issues, LMK.

Resolves #2914, resolves #2802, resolves #2628

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of them working

**Comment by StrikerFRFX** [comment]
Proof of working:
99hEuvWSyX
zvLVMrQd3W
ofUMPC95GQ
7d1vzww9MN
ZkzDuavidw
CSdUOCIH0x
dZcFaM7WGJ
PDiQ7zahQT
NUcZt7M9Na
uRRBmxaumr

**Comment by StrikerFRFX** [comment]
And Vimeo:
cjLf22WBZ9

File: websites/A/Amazon Music/dist/metadata.json [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
   "description": {
     "en": "Amazon Music is a music streaming platform and online music store operated by Amazon.com. Launched in public beta on September 25, 2007, in January 2008 it became the first music store to sell music without digital rights management from the four major music labels, as well as many independents.",
     "zh_CN": "Amazon Music是由Amazon.com经营的音乐媒体平台和在线音乐商店。",
     "zh_TW": "Amazon Music是由Amazon.com經營的音樂串流平台和在線音樂商店。"
   },
   "url": "music.amazon.com",
-  "version": "1.2.5",
-  "logo": "https://i.gyazo.com/16875e5712ac33890f17235d0e46f3be.png",
-  "thumbnail": "https://i.gyazo.com/7f39c3c0b18337d8e6f3bcd0cf962a01.png",
+  "version": "1.2.7",
```
Comment (Bas950) [inline websites/A/Amazon Music/dist/metadata.json:19]: that bump doesnt make sense xD

File: websites/A/AnimeDigitalNetwork/dist/metadata.json [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
   "service": "AnimeDigitalNetwork",
   "description": {
     "en": "Anime Digital Network is the best of the live animated Japan! See all Naruto Shippuden, Fairy Tail, Hunter x Hunter, Blue Exorcist, Code Geass, Vampire Knight, and more.",
     "zh_CN": "AnimeDigitalNetwork是最棒的日本在线动漫网站。"
   },
   "url": "animedigitalnetwork.fr",
-  "version": "1.2.5",
-  "logo": "https://i.gyazo.com/e9fb770cd960ee72ea6e1c58c08113d7.png",
-  "thumbnail": "https://i.gyazo.com/7f6b1357233ab82c23de500417281a15.png",
+  "version": "1.2.7",
```
Comment (Bas950) [inline websites/A/AnimeDigitalNetwork/dist/metadata.json:18]: same why go up 2 versions

File: websites/I/iHeartRadio/dist/metadata.json [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
+  ],
   "service": "iHeartRadio",
   "description": {
     "en": "iHeartRadio is a free broadcast, podcast and streaming radio platform owned by iHeartMedia, Inc. It was founded in April 2008."
   },
   "url": "www.iheart.com",
-  "version": "1.1.6",
-  "logo": "https://i.gyazo.com/5e3a5846ee0601d93b717ac6a689f11c.png",
-  "thumbnail": "https://i.gyazo.com/d624c8a4691282087bb9f50812a6f53a.png",
+  "version": "1.1.8",
```
Comment (Bas950) [inline websites/I/iHeartRadio/dist/metadata.json:17]: here too

File: websites/P/Pokémon Showdown/dist/metadata.json [hunk 4]
```diff
@@ -1,19 +1,28 @@
...
   },
-  "url": "play.pokemonshowdown.com",
-  "version": "1.1.6",
-  "logo": "https://i.gyazo.com/94f265efc7feea6b71d3455c6d5d1b66.png",
-  "thumbnail": "https://i.gyazo.com/308de2d4bcd3d5de5894c136e5a527c5.png",
+  "url": [
+    "pokemonshowdown.com",
+    "play.pokemonshowdown.com"
+  ],
+  "version": "1.1.8",
```
Comment (Bas950) [inline websites/P/Pokémon Showdown/dist/metadata.json:23]: another one

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**